content/docs/command-reference/dag.md

```diff
@@ -69,22 +71,31 @@ $ dvc exp show ...
 
 ## Options
 
-- `--full` - show full DAG that the `target` stage belongs to, instead of
-  showing only its ancestors.
+- `-h`, `--help` - prints the usage/help message, and exit.
 
-- `--dot` - show DAG in
-  [DOT](<https://en.wikipedia.org/wiki/DOT_(graph_description_language)>)
-  format. It can be passed to third party visualization utilities.
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information.
 
 - `-o`, `--outs` - show a DAG of chained dependencies and outputs instead of the
   stages themselves. The graph may be significantly different.
 
-- `-h`, `--help` - prints the usage/help message, and exit.
+- `--full` - show full DAG that the `target` stage belongs to, instead of
+  showing only its ancestors.
 
-- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
-  problems arise, otherwise 1.
+- `--md` - show DAG in `--mermaid` format, wrapped inside a Markdown
+  [fenced code block](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks).
 
-- `-v`, `--verbose` - displays detailed tracing information.
+  This can be used to combine `dvc dag` with
+  [`cml send-comment`](https://cml.dev/doc/ref/send-comment)
+
+- `--mermaid` - show DAG in [Mermaid](https://mermaid-js.github.io) format. It
+  can be passed to third party visualization utilities.
+
+- `--dot` - show DAG in
+  [DOT](<https://en.wikipedia.org/wiki/DOT_(graph_description_language)>)
+  format. It can be passed to third party visualization utilities.
 
 ## Example: Visualize a DVC Pipeline
 
```

Comment:
```suggestion
- `-h`, `--help` - prints the usage/help message, and exit.

- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
  problems arise, otherwise 1.

- `-v`, `--verbose` - displays detailed tracing information.

## Example: Visualize a DVC Pipeline

```